shoebot/data/typography.py

```diff
@@ -141,8 +142,23 @@ def __init__(
         self.font = kwargs.get("font", canvas.fontfile)
         self.fontsize = kwargs.get("fontsize", canvas.fontsize)
         self.align = kwargs.get("align", canvas.align)
-        self.indent = kwargs.get("indent")
         self.lineheight = kwargs.get("lineheight", canvas.lineheight)
+        self.indent = kwargs.get("indent")
```

Comment:
I guess we can't set indent on canvas ?

Reply:
I'm not sure why, maybe it sat there unfinished? I can try putting it back later.

Reply:
I think it can be set as a kwarg to text(), but it is not part of the state variables like fontsize (which have functions to set them)